### PR TITLE
docs(plugins/dynatrace): update doc with install processing.

### PR DESCRIPTION
**Pull request by padupe**
## Hey, I just made a Pull Request!

plugin

Updated [plugin-dynatrace](https://github.com/backstage/backstage/tree/master/plugins/dynatrace) documentation:
- Requirements
- Installation process
- Necessary changes in the front-ent

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by awanlin** [comment]
The `package.json` changes would be to add the `repository` section like this: https://github.com/backstage/backstage/blob/09950bee045a81a5dd0e271c27f6d031cd4c7693/plugins/azure-devops/package.json#L16.

Making sure to update the `directory` to the right value

File: plugins/dynatrace/README.md [hunk 1]
```diff
@@ -17,6 +21,39 @@ The Dynatrace plugin will require the following information, to be used in the c
...
+const serviceEntityPage = (
+  <EntityLayout>
+    [...]
+    <EntityLayout.Route
+      path="/dynatrace"
+      title="Dynatrace"
+      if={isDynatraceAvailable}
+    >
+      <DynatraceTab />
+    </EntityLayout.Route>
```
Comment (awanlin) [inline plugins/dynatrace/README.md:51]: Missed this the last time around, lines 45 to 51 should all begin with the `+` symbol to show they were added

Comment (padupe) [reply plugins/dynatrace/README.md]: Perfect!

**Comment by awanlin** [comment]
LGTM! Now we just need approval from the maintainers 🚀

**Pull request merged**